logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -21,6 +26,14 @@ class UserSessionScope(
     private val messageRepository: MessageRepository
         get() = MessageRepository(authenticatedDataSourceSet.authenticatedNetworkContainer.messageApi)
 
+    private val preyKeyMapper: PreyKeyMapper get() = PreyKeyMapper()
+    private val locationMapper: LocationMapper get() = LocationMapper()
+    private val clientMapper: ClientMapper get() = ClientMapper(preyKeyMapper, locationMapper)
+
+    private val clientRepository: ClientRepository
+        get() = ClientRepository(authenticatedDataSourceSet.authenticatedNetworkContainer.clientApi, clientMapper)
+
+    val clientScope: ClientScope get() = ClientScope(clientRepository)
```

Comment:
Can we rename this to `client`, or rename the other two to `conversationScope` and `messageScope` ?

When using DSL, adding the `scope` suffix sounds a bit redundant.

Maybe it could even be removed from the `sessionScope(userSession) { ... }` too and make it:

```
core.session(userSession) {
    client.register()
}
```

Reply:
make sense 👍🏼
i will change it to client